kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Timestampable.java

```diff
@@ -20,4 +20,6 @@
 
     T usingTimestamps();
 
+    T withoutUsingTimestamps();
```

Comment:
Isn't this redundant and confusing?

I think that:

https://github.com/fabric8io/kubernetes-client/blob/503fd099714e151211f9b3681cd686bdfbee3e33/openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/BuildOperationsImpl.java#L133-L135

will be invoked even if this configuration flag is omitted.

We could do the same for:

https://github.com/fabric8io/kubernetes-client/blob/503fd099714e151211f9b3681cd686bdfbee3e33/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/PodOperationsImpl.java#L160-L162

and completely remove this option.


Is this possible? what do you think?